--- origsrc/lxsession-0.5.1/Makefile.am	2014-10-11 05:18:08.000000000 -0500
+++ src/lxsession-0.5.1/Makefile.am	2014-11-25 15:22:39.448450400 -0600
@@ -42,7 +42,6 @@ EXTRA_DIST = \
 
 if USE_GTK
 bin_PROGRAMS += \
-    lxpolkit/lxpolkit \
     lxsession-db/lxsession-db \
     lxsession-edit/lxsession-edit \
     lxsession-logout/lxsession-logout \
@@ -78,6 +77,10 @@ lxclipboard_lxclipboard_LDADD = \
     $(X11_LIBS) \
     $(NULL)
 
+if HAVE_POLKIT
+bin_PROGRAMS += \
+    lxpolkit/lxpolkit
+
 lxpolkit_lxpolkit_vala_SOURCES = \
     lxpolkit/main.vala \
     $(NULL)
@@ -111,6 +114,7 @@ lxpolkit_lxpolkit_LDADD = \
     $(POLKIT_LIBS) \
     $(X11_LIBS) \
     $(NULL)
+endif # HAVE_POLKIT
 
 lxsession_db_lxsession_db_SOURCES = \
     lxsession-db/main.vala \
--- origsrc/lxsession-0.5.1/configure.ac	2014-10-17 13:11:57.000000000 -0500
+++ src/lxsession-0.5.1/configure.ac	2014-11-25 15:19:39.416589200 -0600
@@ -90,7 +90,7 @@ else
   AC_SUBST(GTK_LIBS)
 
   polkit_modules="polkit-agent-1"
-  PKG_CHECK_MODULES(POLKIT, [$polkit_modules])
+  PKG_CHECK_MODULES(POLKIT, [$polkit_modules], HAVE_POLKIT=yes, HAVE_POLKIT=no)
   AC_SUBST(POLKIT_CFLAGS)
   AC_SUBST(POLKIT_LIBS)
 
@@ -108,6 +108,7 @@ else
 
 fi
 
+AM_CONDITIONAL(HAVE_POLKIT, test "$HAVE_POLKIT" = "yes")
 AM_CONDITIONAL(USE_BUILDIN_CLIPBOARD, test "$use_buildin_clipboard" = "yes")
 AM_CONDITIONAL(USE_BUILDIN_POLKIT, test "$use_buildin_polkit" = "yes")
 if test "$use_buildin_polkit" = "yes"; then
