--- a/Makefile.am	2016-11-27 16:36:02.000000000 -0600
+++ b/Makefile.am	2016-11-29 17:00:21.086227800 -0600
@@ -42,7 +42,6 @@ EXTRA_DIST = \
 
 if USE_GTK
 bin_PROGRAMS += \
-    lxpolkit/lxpolkit \
     lxsession-db/lxsession-db \
     lxsession-edit/lxsession-edit \
     lxsession-logout/lxsession-logout \
@@ -91,6 +90,10 @@ lxclipboard_lxclipboard_CPPFLAGS += $(UN
 lxclipboard_lxclipboard_LDADD += $(UNIQUE_LIBS)
 endif
 
+if HAVE_POLKIT
+bin_PROGRAMS += \
+    lxpolkit/lxpolkit
+
 lxpolkit_lxpolkit_vala_SOURCES = \
     lxpolkit/main.vala \
     $(NULL)
@@ -137,6 +140,7 @@ lxpolkit_lxpolkit_VALAFLAGS += --pkg uni
 lxpolkit_lxpolkit_CPPFLAGS += $(UNIQUE_CFLAGS)
 lxpolkit_lxpolkit_LDADD += $(UNIQUE_LIBS)
 endif
+endif # HAVE_POLKIT
 
 lxsession_db_lxsession_db_SOURCES = \
     lxsession-db/main.vala \
--- a/configure.ac	2016-11-27 16:36:02.000000000 -0600
+++ b/configure.ac	2016-11-29 17:00:21.090216900 -0600
@@ -92,7 +92,7 @@ else
   AC_SUBST(VALA_GTK_LIBS)
 
   polkit_modules="polkit-agent-1"
-  PKG_CHECK_MODULES(POLKIT, [$polkit_modules])
+  PKG_CHECK_MODULES(POLKIT, [$polkit_modules], HAVE_POLKIT=yes, HAVE_POLKIT=no)
   AC_SUBST(POLKIT_CFLAGS)
   AC_SUBST(POLKIT_LIBS)
 
@@ -116,6 +116,7 @@ else
 
 fi
 
+AM_CONDITIONAL(HAVE_POLKIT, test "$HAVE_POLKIT" = "yes")
 AM_CONDITIONAL(USE_BUILDIN_CLIPBOARD, test "$use_buildin_clipboard" = "yes")
 AM_CONDITIONAL(USE_BUILDIN_POLKIT, test "$use_buildin_polkit" = "yes")
 AM_CONDITIONAL(USE_GTK3, test "$enable_gtk3" = "yes")
--- a/data/Makefile.am	2016-11-27 16:36:02.000000000 -0600
+++ b/data/Makefile.am	2016-11-29 17:09:06.945404600 -0600
@@ -10,9 +10,11 @@ desktop_in_files = \
 	lxsession-edit.desktop.in \
 	$(NULL)
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
+if HAVE_POLKIT
 autostartdir = $(sysconfdir)/xdg/autostart
 autostart_in_files = lxpolkit.desktop.in
 autostart_DATA = $(autostart_in_files:.desktop.in=.desktop)
+endif
 @INTLTOOL_DESKTOP_RULE@
 DISTCLEANFILES = \
 	lxsession-default-apps.desktop \
--- a/data/ui/Makefile.am	2016-11-27 16:36:02.000000000 -0600
+++ b/data/ui/Makefile.am	2016-11-30 00:50:57.860207100 -0600
@@ -3,11 +3,15 @@ NULL=
 # GtkBuilder UI definition files
 uidir=$(datadir)/lxsession/ui
 ui_DATA= \
-	lxpolkit.ui \
 	lxsession-default-apps.ui \
 	lxsession-edit.ui \
 	$(NULL)
 
+if HAVE_POLKIT
+ui_DATA += \
+	lxpolkit.ui
+endif
+
 EXTRA_DIST= \
 	$(ui_DATA) \
 	$(NULL)
--- a/man/Makefile.am	2016-12-09 19:53:22.000000000 -0600
+++ b/man/Makefile.am	2016-12-12 17:08:25.540248900 -0600
@@ -7,11 +7,15 @@ man_MANS = \
 	lxsession-default-terminal.1 \
 	lxsession-default-apps.1 \
 	lxsettings-daemon.1 \
-	lxpolkit.1 \
 	lxsession-edit.1 \
 	lxsession-db.1 \
 	lxsession-xdg-autostart.1
 
+if HAVE_POLKIT
+man_MANS += \
+	lxpolkit.1
+endif
+
 man_XMANS = \
 	lxsession.xml \
 	lxsession-logout.xml
