--- a/lxsession-default-apps/combobox.vala	2016-11-27 16:36:02.000000000 -0600
+++ b/lxsession-default-apps/combobox.vala	2016-11-29 18:51:28.414636900 -0600
@@ -647,15 +647,15 @@ namespace LDefaultApps
 
     void save_mime_type(string[] mime_list, string base_name)
     {
-        string mimeapps_list_directory = Path.build_filename(Environment.get_home_dir (),".local", "share", "applications");
-        KeyFile kf_mimeapps =  load_key_conf(mimeapps_list_directory, "mimeapps.list");
-        string mimeapps_list_path = Path.build_filename(mimeapps_list_directory, "mimeapps.list");
+        string mimeapps_list_directory = Path.build_filename(Environment.get_home_dir (),".config");
+        string menu_prefix = Environment.get_variable("XDG_MENU_PREFIX");
+        KeyFile kf_mimeapps =  load_key_conf(mimeapps_list_directory, menu_prefix + "mimeapps.list");
+        string mimeapps_list_path = Path.build_filename(mimeapps_list_directory, menu_prefix + "mimeapps.list");
 
         if (mime_list != null)
         {
             for (int i = 0 ; i < mime_list.length ; i++)
             {
-                kf_mimeapps.set_string("Added Associations", mime_list[i], base_name);
                 kf_mimeapps.set_string("Default Applications", mime_list[i], base_name);
             }
         }
