--- origsrc/lxsession-0.5.3/Makefile.am	2018-04-23 23:08:49.969521900 -0500
+++ src/lxsession-0.5.3/Makefile.am	2018-04-23 23:11:37.478031200 -0500
@@ -471,7 +471,7 @@ lxsession_lxsession_LDADD += \
 
 if USE_ADVANCED_NOTIFICATIONS
 lxsession_lxsession_VALAFLAGS +=     \
-    --pkg appindicator-0.1 \
+    --pkg ayatana-appindicator-0.1 \
     --pkg libnotify \
     -D USE_ADVANCED_NOTIFICATIONS
 
--- origsrc/lxsession-0.5.3/configure.ac	2018-04-23 23:08:49.972521400 -0500
+++ src/lxsession-0.5.3/configure.ac	2018-04-23 23:11:28.895049600 -0500
@@ -123,7 +123,7 @@ AM_CONDITIONAL(USE_GTK3, test "$enable_g
 AM_CONDITIONAL(USE_ADVANCED_NOTIFICATIONS, test "$use_advanced_notifications" = "yes")
 
 if test "$use_advanced_notifications" = "yes"; then
-  PKG_CHECK_MODULES(INDICATORS, [indicator-0.4 >= 0.4.93 appindicator-0.1])
+  PKG_CHECK_MODULES(INDICATORS, [ayatana-appindicator-0.1])
   AC_SUBST(INDICATORS_CFLAGS)
   AC_SUBST(INDICATORS_LIBS)
 
